src/core/client/stream/tabs/Comments/Comment/CommentContainer.tsx

```diff
@@ -342,7 +348,7 @@ export const CommentContainer: FunctionComponent<Props> = ({
     indentLevel === 1 &&
     !!showRemoveAnswered;
 
-  const showAvatar = settings.featureFlags.includes(GQLFEATURE_FLAG.AVATARS);
+  //const showAvatar = settings.featureFlags.includes(GQLFEATURE_FLAG.AVATARS);
```

Comment:
May I ask why this feature flag is commented out? Many of our clients don't want avatars enabled, so we use a feature flag to allow everyone to conditionally enable them.